src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 700;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  --color-dynamic: #3d4e61;
+  --color-border: #e1e7ed;
+}
```

Comment:
The distance between two search bars must be 20px. Add a bottom margin to either the first or second form. Example: .form { margin-bottom: 20px; }

Reply:
please let's leave it as it is, otherwise the tests fail
![form margin 20px code ](https://user-images.githubusercontent.com/121374506/231453778-7ed3bf93-d313-4162-a034-b0d12253f429.jpg)
![form margin 20px bra ](https://user-images.githubusercontent.com/121374506/231453780-d8039306-8df4-4734-90d2-911619d8cd1a.jpg)